pycbc/io/live.py

```diff
@@ -248,21 +251,31 @@ def __init__(self, coinc_ifos, ifos, coinc_results, **kwargs):
         else:
             self.p_astro, self.p_terr = None, None
 
-        # Source probabilities estimation
+        # Source probabilities and hasmassgap estimation
         if 'mc_area_args' in kwargs:
             eff_distances = [sngl.eff_distance for sngl in sngl_inspiral_table]
             self.probabilities = calc_probabilities(coinc_inspiral_row.mchirp,
                                                     coinc_inspiral_row.snr,
                                                     min(eff_distances),
                                                     kwargs['mc_area_args'])
+            kwargs['hasmassgap_args'] = copy.deepcopy(kwargs['mc_area_args'])
+            kwargs['hasmassgap_args']['mass_gap'] = True
+            kwargs['hasmassgap_args']['mass_bdary']['ns_max'] = 3.0
+            kwargs['hasmassgap_args']['mass_bdary']['gap_max'] = 5.0
+            self.hasmassgap = calc_probabilities(
+                                  coinc_inspiral_row.mchirp,
+                                  coinc_inspiral_row.snr,
+                                  min(eff_distances),
+                                  kwargs['hasmassgap_args'])['Mass Gap']
```

Comment:
This looks like Veronica's recent change. Rebase on master to clean this up.

Reply:
I am good at many things, it seems git is not one of them.
I think I've fixed it, but I can't seem to update the PR? Is the correct button available to you?

Reply:
Let's discuss over Slack about this.